backend-issuer/package.json

```diff
@@ -10,7 +10,7 @@
     "build:tsoa": "tsoa spec-and-routes",
     "build:prisma": "prisma generate",
     "build": "npm run build:prisma && npm run build:tsoa && tsc",
-    "start": "node build/index.js",
```

Comment:
went for this solution rather than making the dockerfile copy into a `/build` directory, but I could do that way if preferred. This does mean that `npm start` doesn't work in a dev setup, but I figure you always use `npm run dev` as a dev